### DISCUSSION TITLE
I updated to 0.4.4  but xp12 still crashing

### DISCUSSION THREAD

**Discussion by c777alpha** [kubilus1/autoortho#242] [General]
I updated to 0.4.4  but xp12 still crashing....I attached a copy of auto ortho repeating pattern
![autoortho](https://github.com/kubilus1/autoortho/assets/138440465/e7883e0b-6433-4005-bff7-f6aca51d8135)

please help

**Comment by dpdurst** [comment]
Please only open one issue per problem, this same question was already asked and answer https://github.com/kubilus1/autoortho/issues/243
